packages/cli/src/ceramic-daemon.ts

```diff
@@ -805,10 +831,24 @@ export class CeramicDaemon {
       return
     }
 
-    const indexedModelStreamIDs = await this.ceramic.admin.getIndexedModels()
-    res.json({
-      models: indexedModelStreamIDs.map((modelStreamID) => modelStreamID.toString()),
-    })
+    const withFieldIndices = req.query.withFieldIndices
+    if (withFieldIndices && withFieldIndices.toString().toLowerCase() == 'true') {
+      const indexedModels = await this.ceramic.admin.getIndexedModelsWithFieldIndices()
+      res.json({
+        models: indexedModels.map((idx) => idx.streamID.toString()),
+        indices: indexedModels.map((idx) => {
+          return {
+            streamID: idx.streamID.toString(),
+            indices: idx.indices,
+          }
+        }),
+      })
+    } else {
+      const indexedModels = await this.ceramic.admin.getIndexedModels()
+      res.json({
+        models: indexedModels.map((idx) => idx.toString()),
```

Comment:
if we take my suggestion from above to make the `indices` top-level field name something more general and extensible, then we probably still want to return it here with the object-based output format, just with the `indices` array field omitted.